### ISSUE TITLE
Pip dependency issues upon installation

### ISSUE THREAD

**Issue by lefterav** [DFKI-NLP/diamat#2]
While installing on two different systems, one with a Linux Mint 18.3 64bit and one Ubuntu 16.04 running the `pip install -r requirements.txt` bumped into the following error:

```
ERROR: requests 2.21.0 has requirement urllib3<1.25,>=1.21.1, but you'll have urllib3 1.25.2 which is incompatible.
ERROR: botocore 1.12.143 has requirement urllib3<1.25,>=1.20; python_version >= "3.4", but you'll have urllib3 1.25.2 which is incompatible.
```

I haven't checked how this affects the execution

**Comment by rbtsbg** [comment]
Yes, the errors also occur on my machine now. According to dependencies.txt urllib3 1.21.1 was installed when we ran the experiments but now urllib 1.21.2 is being installed, which apparently causes the collisions you cite. 

Maybe fix the version of urllib3 in requirements.txt?
